patch/kernel/sunxi-next/lime2-add-A20-CAN-support.patch

```diff
@@ -0,0 +1,77 @@
+diff --git a/arch/arm/boot/dts/sun7i-a20-olinuxino-lime2.dts b/arch/arm/boot/dts/sun7i-a20-olinuxino-lime2.dts
+index d5c796c..2f90530 100644
+--- a/arch/arm/boot/dts/sun7i-a20-olinuxino-lime2.dts
++++ b/arch/arm/boot/dts/sun7i-a20-olinuxino-lime2.dts
+@@ -278,3 +278,9 @@
+ 	usb2_vbus-supply = <&reg_usb2_vbus>;
+ 	status = "okay";
+ };
++
++&can0 {
++	pinctrl-names = "default";
++	pinctrl-0 = <&can0_pins_a>;
++	status = "okay";
++};
+diff --git a/arch/arm/boot/dts/sun7i-a20.dtsi b/arch/arm/boot/dts/sun7i-a20.dtsi
+index bd0c476..8b63dde 100644
+--- a/arch/arm/boot/dts/sun7i-a20.dtsi
++++ b/arch/arm/boot/dts/sun7i-a20.dtsi
+@@ -1229,6 +1229,13 @@
+ 				allwinner,pull = <SUN4I_PINCTRL_NO_PULL>;
+ 			};
+ 
++			can0_pins_a: can0@0 {
++				allwinner,pins = "PH20","PH21";
++				allwinner,function = "can";
++				allwinner,drive = <0>;
++				allwinner,pull = <0>;
```

Comment:
Please use symbolic constants here like `SUN4I_PINCTRL_10_MA` and `SUN4I_PINCTRL_NO_PULL`